### PR TITLE
Rewrite Use Import subchapter

### PR DESCRIPTION
**Pull request by Veykril**
Fixes https://github.com/ferrocene/specification/issues/324
Fixes https://github.com/ferrocene/specification/issues/323

### CODE REVIEW

File: src/entities-and-resolution.rst [hunk 1]
```diff
@@ -1006,36 +1010,113 @@ A :t:`use import` brings :t:`entities <entity>` :t:`in scope` within the
...
+A :t:`simple import` is a :t:`use import` that brings all :t:`entities <entity>`
+it refers to into scope, optionally with a different
+:t:`name` than they are declared with by using a :t:`renaming`.
+
 :dp:`fls_v3a6y2ze44v2`
 A :t:`glob import` is a :t:`use import` that brings all :t:`entities <entity>`
-with :t:`public visibility` prefixed by its :t:`simple path prefix` into
-:t:`scope`.
+exported by the :t:`module` or :t:`enum` its :t:`import path prefix` resolves to
+into :t:`scope`.
```
Comment (Dajamante) [inline src/entities-and-resolution.rst:1051]: This sentence is quite hard to understand: "
"A glob import is a use import that brings all entities exported by the module or enum its import path prefix resolves to into scope."
Do you mean it brings all modules pointed to by the import path prefix into scope?

**Review by Dajamante — APPROVED** [review]

File: src/entities-and-resolution.rst (hunk 1 above)
Comment (Veykril) [reply src/entities-and-resolution.rst:1051]: it brings all entities into scope that are exported in the module or enum that the path prefix resolves to, that is
```
foo::bar::*
```
imports everything from the `foo::bar` module (or enum)

Comment (Dajamante) [reply src/entities-and-resolution.rst:1051]: If it is clear to you, then that is good enough.

**Comment by Dajamante** [comment]
bors merge

**Pull request merged**